### PR TITLE
TGP-1967 Create File Ready Page

### PR DESCRIPTION
**Pull request by Ewan-James-Donovan**
There's a lot of TODOs, discussed with amar and friends about file download things. It's okay to have the todo's for now on this ticket

### CODE REVIEW

File: app/views/FileReadyView.scala.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@(fileSizeKilobytes: Int, fileDownloadLink: String, createdDate: String, availableUntil: String)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("fileReady.title"))) {
+
+    <h1 class="govuk-heading-l">@messages("fileReady.heading")</h1>
+
+    <p class="govuk-body">@messages("fileReady.p1", createdDate, availableUntil)</p>
+
+    <p class="govuk-body">
+        <a href="//@fileDownloadLink" class="govuk-link">@messages("fileReady.p2.linkText", fileSizeKilobytes.toString)</a>
```
Comment (Ewan-James-Donovan) [inline app/views/FileReadyView.scala.html:30]: double slash ensures absolute path: `//@fileDownloadLink`

File: app/controllers/FileReadyController.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  override val messagesApi: MessagesApi,
+  identify: IdentifierAction,
+  getData: DataRetrievalAction,
+  requireData: DataRequiredAction,
+  val controllerComponents: MessagesControllerComponents,
+  view: FileReadyView
+) extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
```
Comment (vinoocs) [inline app/controllers/FileReadyController.scala]: may need ProfileAuth action

File: app/views/FileReadyView.scala.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    govukButton: GovukButton,
```
Comment (vinoocs) [inline app/views/FileReadyView.scala.html]: believe govukButton can be removed as it's never used

Comment (Ewan-James-Donovan) [reply app/views/FileReadyView.scala.html]: true!

File: app/controllers/FileReadyController.scala (hunk 2 above)
Comment (Ewan-James-Donovan) [reply app/controllers/FileReadyController.scala]: sure!

**Review by vinoocs — DISMISSED** [review]

**Review by yohan-oc — APPROVED** [review]

**Review by vinoocs — APPROVED** [review]

**Review by mycp98 — APPROVED** [review]

**Pull request merged**